samples/access_token_data.json

```diff
@@ -1,6 +1,7 @@
 {
-  "access_token": "123",
-  "expiry": "123135429034087083",
+  "username": "mshaposhnik+tekton_demo",
+  "access_token": "TPQ8OQSY8CC7SNLKH7T7RFN9G8B8N2ZDNC09M......",
```

Comment:
can we make a separate file with `username` and `access_token` for testing the token endpoint? 
Just do not confuse with optional fields.

Reply:
\+